app/src/main/java/edu/grinnell/grinnell_publications_android/Activities/MainActivity.java

```diff
@@ -43,6 +45,12 @@ protected void onCreate(Bundle savedInstanceState) {
     }
 
     @Override
+    public boolean onCreateOptionsMenu(Menu menu) {
+        getMenuInflater().inflate(R.menu.menu, menu);
```

Comment:
According to the [spec](https://appdev.slack.com/files/sonsooji/F2FNLNFE3/subscription_management_page.png), you should only have the search icon on just the Publications Fragment. This layout inflation should not happen here in `MainActivity.java` . It should be done in `PublicationsFragment.java`